### ISSUE TITLE
Drag error in the example

### ISSUE THREAD

**Issue by int0h** [almende/vis#1300]
I accidentally found a small bug. There's way how to see it:
1) let's go to the http://visjs.org/examples/network/other/manipulation.html 
2) enable edit mode
3) select any edge
4) click "edit edge" button
5) drag  and drop just a single node
(mouse down on any node and mouse up while editing an edge)

All this throws an uncaught error in 40713 line;

I got rid of this error via adding extra condition: 
```javascript
...
          if (overlappingNodeIds[i] !== this.selectedControlNode.id) {
...
```
becomes 
```javascript
...
        if (!this.selectedControlNode || overlappingNodeIds[i] !== this.selectedControlNode.id) {
...
```

But I didn't dive into your code at all. I don't know what exactly wrong here. I just fixed the uncaught error in my case. I hope this message will help you

**Comment by AlexDM0** [comment]
Hi,

Thanks for the heads up. Guess I overly assumed people dragging the control nodes once that mode is active. I'll get this fixed.

Regards

**Comment by AlexDM0** [comment]
Hi,

I've pushed a fix for this to develop. We'll add it to the next release.

Thanks for letting us know!

Cheers

**Comment by AlexDM0** [comment]
Released in v4.9.0
